apps/editing-toolkit/editing-toolkit-plugin/common/index.php

```diff
@@ -15,19 +15,7 @@
  * Register data stores that may be useful for a variety of concerns
  */
 function register_data_stores() {
-	$path         = plugin_dir_path( __FILE__ ) . 'dist/data_stores.js';
-	$asset_file   = plugin_dir_path( __FILE__ ) . 'dist/data-stores.asset.php';
-	$asset        = file_exists( $asset_file ) ? require $asset_file : null;
-	$dependencies = isset( $asset['dependencies'] ) ? $asset['dependencies'] : array();
-	$version      = isset( $asset['version'] ) ? $asset['version'] : filemtime( $path );
-
-	wp_register_script(
-		'a8c-fse-common-data-stores',
```

Comment:
If this handle is changing names, all of these need to be updated:

https://github.com/Automattic/wp-calypso/blob/61acc957638fc7009632af4c0c058aa6bc0e82b7/apps/editing-toolkit/typings/fse/index.d.ts#L3
https://github.com/Automattic/wp-calypso/blob/61acc957638fc7009632af4c0c058aa6bc0e82b7/apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/stores/index.ts#L4
https://github.com/Automattic/wp-calypso/blob/ac77fdc5aea1f5b038918dbe3eefdaf80cceaf89/apps/editing-toolkit/webpack.config.js#L85-L89
https://github.com/Automattic/wp-calypso/blob/61acc957638fc7009632af4c0c058aa6bc0e82b7/apps/editing-toolkit/editing-toolkit-plugin/editor-gutenboarding-launch/index.ts#L9

As well as the [docs here](https://github.com/Automattic/wp-calypso/blob/1deac6cbd03d731759a30d40e6ffa480830231d7/apps/editing-toolkit/README.md#shared-wordpress-scripts).

It would be nice to handle this in the build. Maybe shared functionality, things that register scripts but don't enqueue them to enable features, should live in a specific module and webpack should know how to externalize them. At the moment, I believe that's only data-stores.

Reply:
I've made these changes in #50511 (366d13e7a7c7635ef760b79284eb08f598e544ab). I'll keep this in the back of my mind; I do agree that it could be automated further